app/models/miq_request_workflow.rb

```diff
@@ -33,6 +33,19 @@ def self.all_encrypted_options_fields
     descendants.flat_map(&:encrypted_options_fields).uniq
   end
 
+  def self.update_requester_from_parameters(data, user)
+    return user if data[:user_name].blank?
+    new_user = User.lookup_by_identity(data[:user_name])
+
+    unless new_user
+      _log.error "requested not changed to <#{data[:user_name]}> due to a lookup failure"
+      raise ActiveRecord::RecordNotFound
+    end
+
+    _log.warn "requested changed to <#{new_user.userid}>"
+    new_user
```

Comment:
@gmcculloug @tinaafitz 
Should this be setting the User.current_user to new_user so that the newly created request can have the same tenant id.
Also should we be checking if the user is part of the same 'parent' tenant.

@tinaafitz is working on an issue similar to this for Service Provisioning